### DISCUSSION TITLE
Condivisione di una o più entità con i clienti in HomeAssistant

### DISCUSSION THREAD

**Discussion by solproget** [indomus/forum#2505] [GENERALE - Discussioni generiche]
Buongiorno a tutti,
chiedo se è possibile creare un utente che abbia accesso ad una sola entità di modo che posso fornire un accesso ad i clienti del mio b&b. Nello specifico vorrei fornire il comando per aprire il cancello della struttura. Attualmente se creo un utente riesco a fornire questo limitando la visibilità sulla plancia ad una sola entità però sul menù laterale ci sono altre che non vorrei fossero visibili come lo storico, la mappa, la plancia energia i media e quant' altro.
Ringrazio tutti dell' aiuto.

**Comment by snakuzzo** [comment]
In realtà puoi limitare anche quello che hai in spalla sx.
Esempio...

![image](https://github.com/indomus/forum/assets/7205371/a8fa85dc-6e09-455d-b291-b5a438c8c0ad)

**Comment by solproget** [comment]
![01](https://github.com/indomus/forum/assets/77103958/d24e8c20-b1e8-4258-8937-82d56def14f1)
Queste schede non riesco a nasconderle. Scusate l' ignoranza.

**Comment by idalgo73** [comment]
Buongiorno, anch'io ho lo stesso problema e non so come fare.......

**Comment by gitmed1** [comment]
Ciao a tutti, volevo condividere la soluzione che ho trovato. Se state cercando ancora di risolvere questo problema. Crea un automazione in home assistant e in "quando" inserisci webhook. Di fianco c'è la rotella di configurazione e flagga "get" e se non c'è copertura WiFi nel punto in cui vuoi far aprire il cancello disabilita "accessibile solo dalla rete locale". Dopo di che in "allora" attiva l'azione che ti interessa(apri il cancello, accendi la luce etc..) fatto ciò di fianco al webhook c'è il simbolo del copia, cliccalo e incolla il comando in un browser (io ho usato Chrome) in alto a destra ci sono i 3 puntini e clicca su "aggiungi alla schermata home" dagli il nome che preferisci ed ecco fatto... Ora puoi inviare l'icona a chi vuoi e far aprire il cancello ai tuoi clienti (per il tuo caso specifico meglio se prende il WiFi in questo modo potranno utilizzarlo solo chi è lì alla struttura)

**Reply by solproget** [reply]
Fantastico, funziona tutto correttamente. Grazie per la dritta.
